src/views/product-downloads-view/components/featured-tutorials-section/featured-tutorials-section.module.css

```diff
@@ -16,11 +16,11 @@ out.
   grid-column-gap: 24px;
   grid-row-gap: 24px;
 
-  @media only screen and (min-width: 729px) and (max-width: 1000px) {
+  @media (--dev-dot-tablet) {
```

Comment:
Made these changes to demo that this change _seems_ to work. Also checked the `/tutorials` view, it seems that `TutorialsMeta` retains the custom-media queries it was using (which were already working).